File: Examples/Java/SDK/src/main/java/com/aspose/imaging/cloud/examples/manipulate/RotateFlipImageOnStorage.java
package com.aspose.imaging.cloud.examples.manipulate;

import com.aspose.imaging.cloud.examples.Utils;
import java.io.IOException;
import java.nio.file.Files;
import java.nio.file.Path;
import java.nio.file.StandardCopyOption;

public class RotateFlipImageOnStorage {

    public static void main(String... args) throws IOException {
        String input = "Sample1.png";
        String output = "Sample2.jpg";
        Path inputFile = Utils.getPath(RotateFlipImageOnStorage.class, input);
        Path outputFile = Utils.getPath(RotateFlipImageOnStorage.class, output);

        String format = "jpg";
        String method = "Rotate180FlipX";
        String outPath = "";

        Utils.getStorageSdk().PutCreate(
                input,
                null,
                Utils.STORAGE,
                inputFile.toFile()
        );

        com.aspose.imaging.model.ResponseMessage apiResponse
                = Utils.getImagingSdk().GetImageRotateFlip(
                        input,
                        format,
                        method,
                        outPath,
                        Utils.FOLDER,
                        Utils.STORAGE
                );

        Files.copy(apiResponse.getInputStream(), outputFile, StandardCopyOption.REPLACE_EXISTING);

    }
}

File: Examples/Node.js/SDK/manipulate/RotateFlipImageOnStorage.js
var fs = require('fs'),
    utils = require('../utils.js'),
    assert = require('assert'),
    image = utils.imagingApi;
    storage = utils.storageApi;

var inputName = "sample1.png";
var inputPath = utils.getPath(__filename, inputName);

var outputName = "nodejsSDK.output.jpg";
var outputPath = utils.getPath(__filename, outputName);

var format = "jpg";
var method = "Rotate90FlipX";
var Storage = null;
var folder = null;

storage.PutCreate(inputName, null, Storage, inputPath, function (responseMessage) {
    image.GetImageRotateFlip(inputName, format, method, null, folder, Storage, function(responseMessage) {

        storage.GetDownload(inputName, null, null, function (responseMessage) {
            var writeStream = fs.createWriteStream(outputPath);
            writeStream.write(responseMessage.body);
        });
    });
});